src/server/pfs/server/driver.go

```diff
@@ -429,6 +429,11 @@ func (d *driver) createProject(ctx context.Context, req *pfs.CreateProjectReques
 		return errors.Wrapf(err, "invalid project name")
 	}
 	return d.env.TxnEnv.WithWriteContext(ctx, func(txnCtx *txncontext.TransactionContext) error {
+		whoAmI, err := txnCtx.WhoAmI()
```

Comment:
Since the authentication check is already happening at the interceptor, it would be cleaner to include this into the Create Role binding block:

`if whoAmI, err := txnCtx.WhoAmI(); err != nil {
....
} else if err !errors.Is(err, auth.ErrNotActivated) {
  return errors.Wrapf(err, "error authenticating (must log in to create project)")
}`